projects/transformers/experiments/finetuning.py

```diff
@@ -51,10 +93,19 @@
     do_train=True,
     do_eval=True,
     do_predict=True,
+    eval_steps=10,
+    evaluation_strategy="steps",
+    load_best_model_at_end=True,
     per_device_train_batch_size=32,
     per_device_eval_batch_size=32,
     learning_rate=2e-5,
-    num_train_epochs=3,
+    warmup_ratio=0.1,
```

Comment:
No action here, just a comment: that is the first I see it defined as a warmup_ratio, that is interesting. Previously I've only seen it defined as an absolute number (warmup_steps) in HF examples.